_src/pattern-library/sass/components/_forms.scss

```diff
@@ -0,0 +1,473 @@
+// ------------------------------
+// edX Pattern Library: Components - Forms
+
+// About: Contains base styling for forms
+
+// #GLOBAL
+// #INPUT TEXT
+// #INPUT RADIO/CHECKBOX
+// #SELECT
+// #TEXTAREA
+// #SLIDERS
+// #PROGRESS
+
+// ----------------------------
+// #GLOBAL
+// ----------------------------
+.field-group {
+    @include clearfix;
+    margin-bottom: spacing-vertical(mid-small);
+
+    .field-group-hd {
+        margin-bottom: spacing-vertical(small);
+        font-size: font-size(mid-large);
+    }
+
+    .field {
+        margin-bottom: spacing-vertical(base);
+
+        .helper-text {
+            font-size: font-size(small);
+            color: $field-tip-color;
+        }
+    }
+
+    &.form-group {
```

Comment:
The compound/nested selector here worries me. What if someone using the PL just used one of these classes? Do we need to separate out rules so they are not dependent on each other? If not, this class' name should reflect its dependency on ``.field-group`` according to [our naming conventions](https://github.com/edx/ux-pattern-library/wiki/Styleguide:-Sass-&-CSS#ui-naming-conventions).